tests/data/say-formatting/litexa/main.litexa

```diff
@@ -66,6 +66,12 @@ launch
     say "{'There'} should be a space
       between 'there' and 'should'."
 
+  when EscapesTest
+    say "\<amazon:effect name='whispered'>Hi. I am not a human.\</amazon:effect>"
+    say "That will be \$400 please."
+    # say "Oh, you mean \"that\" thing." # TODO: fix. This should not result in a syntax error
```

Comment:
Would write up a GitHub issue for this.

Reply:
https://github.com/alexa-games/litexa/issues/116